testsuite/scripts/sclang_test_runner.scd

```diff
@@ -144,6 +167,10 @@ if(File.exists(~testProto).not) {
 	// ~writeResult.(tests, file);
 	~verboseSetup.if({"After exclude: ".post; tests.do(_.postln)});
 
+  tests = ~selectThisTestClump.(tests, ~thisClump, ~nClumps);
```

Comment:
Is the indentation off here? Looks strange on GH.

Reply:
Yes, please fix the indentation @frenchy64